ex4/server/routes/api.js

```diff
@@ -0,0 +1,39 @@
+const ItemManager = require('../services/item_manager');
+
+async function getTodos(req, res) {
+	let data = await ItemManager.getTodos();
+	if (!data) data = [];
+
+	res.header('Access-Control-Allow-Origin', '*');
+	res.status(200).json(data);
+}
+
+async function createTodo(req, res) {
+	const data = await ItemManager.addTodo(req.body);
```

Comment:
its a bad practice to send all the request body like this, this way the service can't expect what it will get,
the controller should do the validation and the service functions should have clear API